conda/recipe/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', 'unknown').split('.')[:2]) %}
+{% set py_version=environ.get('CONDA_PY', 35) %}
+package:
+  name: meta_nvstrings
```

Comment:
@jakirkham if I named this the same as one of the outputs it would override some of the properties set below. Is there a best practice of what this should be set to when I don't want a top level project?

Reply:
Talked offline and as long as we don't impose any requirements this name doesn't really matter.